service/header/subscription.go

```diff
@@ -30,15 +30,14 @@ func newSubscription(topic *pubsub.Topic) (*subscription, error) {
 func (s *subscription) NextHeader(ctx context.Context) (*ExtendedHeader, error) {
 	msg, err := s.subscription.Next(ctx)
 	if err != nil {
-		log.Errorw("reading next message from subscription", "err", err.Error())
```

Comment:
Note for @renaynay. There is no value in logging the error here as the only types of errors we will see here are context-related. And users of NextHeader should handle them accordingly as we do in DASer.